cmd/tg_bot/internal/environment.go

```diff
@@ -154,3 +163,137 @@ func (tgEnv *TelegramBotEnvironment) NodesListMessage(urls []string) (string, er
 	}
 	return w.String(), nil
 }
+
+func (tgEnv *TelegramBotEnvironment) SubscribeToAllAlerts() error {
+
+	for alertType, alertName := range entities.AlertTypes {
+		if tgEnv.IsAlreadySubscribed(alertType) {
+			return errors.Errorf("failed to subscribe to %s, already subscribed to it", alertName)
+		}
+		err := tgEnv.pubSubSocket.SetOption(mangos.OptionSubscribe, []byte{byte(alertType)})
+		if err != nil {
+			return err
+		}
+		tgEnv.mu.Lock()
+		tgEnv.subscriptions[alertType] = alertName
+		tgEnv.mu.Unlock()
+		log.Printf("Subscribed to %s", alertName)
+	}
+
+	return nil
+}
+
+func (tgEnv *TelegramBotEnvironment) SubscribeToAlert(alertType entities.AlertType) error {
+	alertName, ok := entities.AlertTypes[alertType] // check if such an alert exists
+	if !ok {
+		return errors.New("failed to subscribe to alert, unkown alert type")
+	}
+
+	if tgEnv.IsAlreadySubscribed(alertType) {
+		return errors.Errorf("failed to subscribe to %s, already subscribed to it", alertName)
+	}
```

Comment:
Maybe just return in this case?

Reply:
No. I decided that before using this method we must check whether the user is already subscribed to it or not. I do not want this method to do nothing in the wrong usage scenario

Reply:
Okay